packages/nextjs/src/nextRouteHandler.ts

```
@@ -0,0 +1,173 @@
+import {
+  createRequestHandler,
+  FlowgladServer,
+} from '@flowglad/nextjs';
+import {
+  createRequestHandler,
```

Biome (2.1.2)

[error] Shouldn't redeclare 'createRequestHandler'. Consider to delete it or rename it.

'createRequestHandler' is defined here:


(lint/suspicious/noRedeclare)